controllers/team.controller.js

```diff
@@ -19,9 +19,27 @@ const Constants = {
  * @description Returns the JSON of team object located in req.body.team
  */
 function showTeam(req, res) {
+    const teamData = req.body.team.toJSON();
+    delete teamData.members;
+
+    const memberNames = [];
+    for (const member of req.body.teamMembers) {
+        const strippedMemberJSON = member.toStrippedJSON();
+
+        const memberName = {
+            "firstName": strippedMemberJSON.firstName,
+            "lastName": strippedMemberJSON.lastName,
+        };
+
+        memberNames.push(memberName);
+    }
+
     return res.status(200).json({
         message: Constants.Success.TEAM_READ,
-        data: req.body.team.toJSON()
+        data: {
+            team: teamData,
+            members: memberNames,
```

Comment:
`members` -> `memberNames`?

Reply:
Right now there is no need to display full member information. We have discussed changing routes so that formatting can be done by toJSON() depending on the route context. At that point, I imagine data to be {team: teamData, members: members} with information within members differing based on data retrieval context. However, until that point I want the controller variables to clearly show that we are always only revealing the names of the members.